README.md

```diff
@@ -25,6 +20,48 @@ View the [application examples](core/examples) to explore Luigi's features.
 
 For details, see [Luigi documentation](docs/README.md).
 
+## Browser support
+
+If you want to support additionally legacy browsers like Internet Explorer 11 in your application, you just need to update your Luigi imports as follows:
+### Luigi Core
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel='stylesheet' href='core/public/luigi-legacy.css'>
+    <!-- <link rel='stylesheet' href='core/public/luigi.css'> -->
+  </head>
+  <body>
+    <script src="/luigi-core/luigi-legacy.js"></script>
```

Comment:
How about adding both and loading them conditionally? 

    <script type="module" src="luigi.js"></script>
    <script nomodule src="luigi-legacy.js"></script>

Further read on that topic. https://jakearchibald.com/2017/es-modules-in-browsers/  
And we need to decide if we want to support Safari 10.1, which would require a small polyfill to support `nomodule`.

For CSS: If someone wants to support legacy, the legacy.css is fine in any case. Js would be better to serve conditionally so new browsers can benefit.

Reply:
Good point, I will add it.

Reply:
done